### ISSUE TITLE
`MIPS Constant Reference Analyzer` getting stuck

### ISSUE THREAD

**Issue by mumbel** [NationalSecurityAgency/ghidra#5401]
**Describe the bug**
After coming across #5400 , I tried auto-analysis on another image I've been working on and it got stuck the same way but in the constant analyzer.  Not sure if its the same core issue or not, so making a new ticket

MIPS:BE:64:64-32addr (1.7)

**Attachments**
immediate  [constant1.txt](https://github.com/NationalSecurityAgency/ghidra/files/11608253/constant1.txt)
one min  [constant2.txt](https://github.com/NationalSecurityAgency/ghidra/files/11608256/constant2.txt)
five min  [constant3.txt](https://github.com/NationalSecurityAgency/ghidra/files/11608260/constant3.txt)

**Environment (please complete the following information):**
 - OS: 22.04
 - Java Version: 17.0.7
 - Ghidra Version: 10.3
 - Ghidra Origin: official

**Comment by emteere** [comment]
Are you using inline on any functions?

**Comment by mumbel** [comment]
@emteere   sorry I missed the edit.

for reference this is the MIPS inline function:
```
         03 e0 00 08     jr         ra
         03 e0 10 2d     _daddu     v0,ra,zero
```

on master when this is the only inline, it seems to be getting past the lock up spot (unless the ordering changed, I let it get a couple thousands higher)
verified if i go back to 10.3 and add the inline back it locks up at the same index as before
and 10.3 in eclipse I have two threads it looks like and both startAddr's are processing functions that do call this function

**Comment by mumbel** [comment]
10.3.1_patch looks good w/ inline

**Comment by ryanmkurtz** [comment]
Fixed by bf8d314ddb5822a1b1663dca7f5c7ca3052c2d9e
